src/main/webapp/app/exercises/shared/team/team-participation-table/team-participation-table.component.ts

```diff
@@ -134,6 +134,8 @@ export class TeamParticipationTableComponent implements OnInit {
      * @param submission Either submission or 'new'
      */
     getAssessmentLink(exercise: Exercise, participation: Participation, submission: Submission | 'new'): string[] {
+        participation ??= {};
+        submission ??= 'new';
         const submissionUrlParameter: number | 'new' = submission === 'new' ? 'new' : submission.id!;
```

Comment:
```suggestion
        const submissionUrlParameter: number | 'new' = submission === 'new' || submission === undefined ? 'new' : submission.id!;
```

Reply:
submission will have the value `null` not `undefined`